code/index.html

```diff
@@ -1,30 +1,110 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Jeans Site</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Lobster&family=Varela&family=Quicksand:wght@700&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <section class="background-img" alt="jeans on hangers">
+
+    <!-- HEADER AND NAV-->
+
+    <header class="header">
+      <a href="#">
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </a>
+
+      <div class="content">
+        <a href="#">About us</a>
+        <a href="#">Partners</a>
+        <a href="#">Events</a>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="nav-sign-in">
+        <a href="#">Sign in</a>
       </div>
+    </header>
+
+    <!-- LOGO -->
+
+    <div class="logo">
+      <h1 class="name">Jeans</h1>
+    </div>
+
+    <!-- INFO -->
+
+    <section class="info">
+      <h2>Find the best fit for you!</h2>
+      <p>We set you up with one of our denim experts and help you find the styles you're craving and the perfect fit for
+        you.</p>
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+
+    <!-- SIGN UP BUTTON -->
+
+    <div class="signup-container">
+      <a href="signup.html" class="sign-up" target="_self">
```

Comment:
No need to add `target` attribute if its value is `_self` - it's the default one, so we can just omit whole attribute :)